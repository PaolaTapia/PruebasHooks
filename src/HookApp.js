import React from 'react';

function HookApp() {
  return (
    <h1>
      Hola Mundo!
    </h1>
  );
}

export default HookApp;
