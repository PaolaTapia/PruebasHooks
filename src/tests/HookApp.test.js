import React, {Component} from 'react';
import {shallow} from 'enzyme';
import toJson from 'enzyme-to-json';
import HookApp from '../HookApp';

describe('pruebas en HookApp', () => {
test('renders correctly HookApp', () => {
     const wrapper = shallow(
       <HookApp />
     );    
     expect(toJson(wrapper)).toMatchSnapshot();
   });
     
})

 
