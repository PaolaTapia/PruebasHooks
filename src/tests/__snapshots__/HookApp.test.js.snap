// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pruebas en HookApp renders correctly HookApp 1`] = `
<h1>
  Hola Mundo!
</h1>
`;
